tests/test_qubit_device.py

```diff
@@ -66,8 +66,9 @@ def mock_qubit_device_extract_stats(monkeypatch):
         m.setattr(QubitDevice, "expval", lambda self, x: 0)
         m.setattr(QubitDevice, "var", lambda self, x: 0)
         m.setattr(QubitDevice, "sample", lambda self, x: 0)
+        m.setattr(QubitDevice, "state", 0)
         m.setattr(
-            QubitDevice, "probability", lambda self, wires=None: 0 if wires is None else wires
+            QubitDevice, "probability", lambda self, wires=None: 0
```

Comment:
I wasn't sure why we originally had ` if wires is None else wires`? But the test wouldn't work with it (possibly due to the wires refactor)